lib/ansible/modules/user.py

```diff
@@ -268,9 +268,10 @@
             - Requires O(local) is omitted or V(False).
         type: str
         version_added: "2.12"
-    account_inactive:
+    password_expire_account_disable:
         description:
             - Number of days after a password expires until the account is disabled.
+            - Currently not supported on FreeBSD.
```

Comment:
probably not osx, solaris, aix either nor any distro not using usermod, not sure if lusermod implements either nor busybox, nor adduser. It is probably easiser to state which ones DO support it.

Reply:
`usermod`, `adduser` and `useradd` all got the `-f, --inactive` option, according to the man pages atleast.
I could test  by removing the conditional in the integration test and see what blows up, since I don't have access to osx, solaris, aix etc.